lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -35,20 +36,22 @@ class CampaignSlide extends React.Component {
           <div className="wrapper">
             {this.renderHeader()}
 
-            {this.campaigns.map(campaign => {
-              // Fix for local files / data missing by preventing null error
-              const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
+            <div className="container__cards">
```

Comment:
isn't this just a triad gallery? http://forge.dosomething.org/#gallery-triad

If you use the triad gallery pattern, you probs won't need the `-signup` class to define the widths etc, and it's already set w/ specifics for responsively working on all device sizes.